nextflow.config

```diff
@@ -50,6 +50,12 @@ params {
     skip_quast                           = false
     skip_prodigal                        = false
 
+    // virus identification options
+    run_genomad                          = false
+    genomad_db                           = null
+    genomad_min_score                    = 0.7
+    genomad_splits                       = 1
```

Comment:
Those params are not documented in `nextflow_schema.json`?

Reply:
I forgot to update the schema after editing @PhilPalmer's code. Thank you for catching this! Schema has been updated and I opened a PR to @PhilPalmer's repo.

Reply:
Thanks!